README.md

```diff
@@ -364,16 +364,21 @@ NAME                                        STATUS   VOLUME
 persistentvolumeclaim/m2-data-spring-boot   Bound    pvc-dab00dfe-a2f6-11e9-98d1-08002798bb5f   1Gi        RWO            standard       4m18s
 ```
 
-Package your Java Application `mvn package` and push the uber java file.
+Package your Java Application `mvn package` and push the `uber` java file.
 ```bash
 kubectl cp target/my-component-1.0-SNAPSHOT.jar POD_NAME:/deployments/my-component-1.0-SNAPSHOT -n demo
 ```
 
+**Remark**: Uou can get the pod name or pod id using this command : `kubectl get pods -l component_cr=spring-boot -o name` where
+you pass as `component_cr` label, the component name. Remove the `pod/` prefix from the name. E.g: `pod/spring-boot-747995b4db-hqxhd` -> `spring-boot-747995b4db-hqxhd`
+
 Start your application within the pod
 ```bash
-kubectl exec POD_NAME -n demo /var/lib/supervisord/bin/supervisord ctl start run-cmd
+kubectl exec POD_NAME -n demo /var/lib/supervisord/bin/supervisord ctl start run
 ```
 
+**Important**: We invite you to use our [`Hal` companion tool](https://github.com/halkyonio/hal#2-deploy-the-component) as it will create and push the code source or binary without to be worry about the kubectl command syntax ;-)
```

Comment:
without to be worry -> without having to worry

Reply:
Will fix